### PR TITLE
Terminology rework 2

### PR DESCRIPTION
**Pull request by mcr**
This reworks the pull request: #164 using the latest text.
Commit 1 - move all the text.
Commit 2 - add produces/consumes.
Commit 3 - move the text into categories.

Reviewing the commits individually is probably better.

### CODE REVIEW

**Comment by mcr** [comment]
Please review @SimonFrost-Arm

**Review by SimonFrost-Arm — APPROVED** [review]

File: draft-ietf-rats-architecture.md [hunk 1]
```diff
@@ -507,42 +507,76 @@ In this situation, the trust model described in {{trustmodel}} is also suitable
...
-Appraisal Policy for Attestation Results:
+: A role performed by an entity (typically a device) whose Evidence must be appraised in order to infer the extent to which the Attester is considered trustworthy, such as when deciding whether it is authorized to perform some operation.
+Produces: Evidence.
 
-: A set of rules that direct how a Relying Party uses the Attestation Results regarding an Attester generated by the Verifiers. Compare /security policy/ in {{RFC4949}}
+Relying Party:
 
-Attestation Result:
+: A role performed by an entity that depends on the validity of information about an Attester, for purposes of reliably applying application specific actions.  Compare /relying party/ in {{RFC4949}}
+Negotiates: the relationship between an Attester and a Verifier.
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md:520]: No idea what this means
```suggestion
```

File: draft-ietf-rats-architecture.md [hunk 2]
```diff
@@ -507,42 +507,76 @@ In this situation, the trust model described in {{trustmodel}} is also suitable
...
 
-Appraisal Policy for Attestation Results:
+: A role performed by an entity (typically a device) whose Evidence must be appraised in order to infer the extent to which the Attester is considered trustworthy, such as when deciding whether it is authorized to perform some operation.
+Produces: Evidence.
 
-: A set of rules that direct how a Relying Party uses the Attestation Results regarding an Attester generated by the Verifiers. Compare /security policy/ in {{RFC4949}}
+Relying Party:
 
-Attestation Result:
+: A role performed by an entity that depends on the validity of information about an Attester, for purposes of reliably applying application specific actions.  Compare /relying party/ in {{RFC4949}}
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md:519]:
```suggestion
: A role performed by an entity that depends on the validity of information about an Attester, for purposes of reliably applying application specific actions.  Compare /relying party/ in {{RFC4949}}.
```
If we put punctuation at end of other definitions, be consistent here too.

File: draft-ietf-rats-architecture.md [hunk 3]
```diff
@@ -507,42 +507,76 @@ In this situation, the trust model described in {{trustmodel}} is also suitable
...
+Reference Value Provider:
+
+: An entity (typically a manufacturer) whose Reference Values help Verifiers appraise Evidence to  determine if acceptable known claims have been recorded by the Attester.
+Produces: Reference Values.
+
+## Artifacts
+
+Claim:
+: A piece of asserted information, often in the form of a name/value pair. Claims make up the usual structure of Evidence.
+(Compare /claim/ in {{RFC7519}})
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md:554]:
```suggestion
Compare /claim/ in {{RFC7519}}.
```
Use punctuation consistently between here, line 519, and line 571

File: draft-ietf-rats-architecture.md [hunk 4]
```diff
@@ -552,23 +586,21 @@ referred to in other documents as known-good values, golden measurements,
...
+Used By: Verifier;
+Produced By: Reference Value Provider.
 
-Relying Party:
-
-: A role performed by an entity that depends on the validity of information about an Attester, for purposes of reliably applying application specific actions.  Compare /relying party/ in {{RFC4949}}
+--- following have not been sorted:
 
-Relying Party Owner:
+Endorsement:
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md:594]: This definition should move up to line 580

File: draft-ietf-rats-architecture.md [hunk 5]
```diff
@@ -552,23 +586,21 @@ referred to in other documents as known-good values, golden measurements,
...
 
-: An entity (typically an administrator), that is authorized to configure Appraisal Policy for Attestation Results in a Relying Party
+: A secure statement that an Endorser vouches for the integrity of an Attester's various capabilities such as Claims collection and Evidence signing
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md:597]:
```suggestion
Used By: Verifier;
Produced By: Endorser
```

File: draft-ietf-rats-architecture.md [hunk 6]
```diff
@@ -552,23 +586,21 @@ referred to in other documents as known-good values, golden measurements,
...
 
 Verifier:
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md:598]: This is already up at line 523

**Review by dthaler — CHANGES_REQUESTED** [review]

**Pull request merged**